dev/shim/dune

```diff
@@ -67,9 +67,7 @@
   ; there is no problem with the other shims since they don't depend on optional build products
   %{project_root}/ide/coqide/coqide_main.exe
 
-  %{bin:coqqueryworker.opt}
-  %{bin:coqtacticworker.opt}
-  %{bin:coqproofworker.opt}
+  %{bin:coqworker.opt}
   %{project_root}/theories/Init/Prelude.vo
```

Comment:
Can you also add the coqworker dep to the coqc shim?

Reply:
May still add some latency so I prefer to do in separate PR.